link_shortener/core/filter.py

```diff
@@ -0,0 +1,23 @@
+'''
+Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
+Licensed under the MIT (Expat) License (see LICENSE in Documentation).
+'''
+
+
+def filter_links(data, filters):
+    owners = set(row['owner'] for row in data)
+
+    if filters['owner']:
+        data = [row for row in data if row['owner'] == filters['owner']]
+
+    if filters['search']:
+        data = [row for row in data if filters['search'] in row['endpoint']]
+
+    return {"owners": owners, "data": data}
+
+
+def define_active(filters):
+    if filters['is_active'] == 'false':
```

Comment:
My to posíláme jako string?

Reply:
Filtery posielame takto:
http://localhost:8000/links/all?is_active=false&owner=vojtech.janousek@applifting.cz&search=k
a na BE ich dostavame takto:
![image](https://user-images.githubusercontent.com/71313106/93340923-c759f680-f82d-11ea-9a96-4aee91516e60.png)